cmd/gcp/main.go

```diff
@@ -63,6 +63,7 @@ var (
 	rejectExtensions           = flag.String("reject_extension", "", "A list of X.509 extension OIDs, in dotted string form (e.g. '2.3.4.5') which, if present, should cause submissions to be rejected.")
 	signerPublicKeySecretName  = flag.String("signer_public_key_secret_name", "", "Public key secret name for checkpoints and SCTs signer. Format: projects/{projectId}/secrets/{secretName}/versions/{secretVersion}.")
 	signerPrivateKeySecretName = flag.String("signer_private_key_secret_name", "", "Private key secret name for checkpoints and SCTs signer. Format: projects/{projectId}/secrets/{secretName}/versions/{secretVersion}.")
+	checkpointInterval         = flag.Duration("checkpoint_interval", -1*time.Second, fmt.Sprintf("Interval between checkpoint publishing. If unset, the Tessera default value is %s. Minimum value is 1200ms.", tessera.DefaultCheckpointInterval))
```

Comment:
Did you mean to leave %s?

Reply:
The minimum value `1200ms` comes from Tessera GCP storage implementation. However, that's not an exported const.

https://github.com/transparency-dev/tessera/blob/29298ee9f33e6513a77b117f73d9d6ed6538865a/storage/gcp/gcp.go#L68-L72

Reply:
Ah! searched for 1200ms, not 1200...

Reply:
I wouldn't say anything about the minimum here - if it changes in the storage impl, this will be out of date with no way to know.  Better to just let the user choose a bad value (which they can do anyway), and get told it's too low by the code which enforces the minimum.